README.md

```diff
@@ -78,6 +78,11 @@ see [Migrate from the SignalFx Java Agent](https://docs.splunk.com/Observability
 <!--DEV_DOCS_WARNING_START-->
 The following documentation refers to the in-development version of `splunk-otel-java`. Docs for the latest version ([v1.14.0](https://github.com/signalfx/splunk-otel-java/releases/latest)) can be found [here](https://github.com/signalfx/splunk-otel-java/blob/v1.14.0/README.md).
 
+---
+<!--DEV_DOCS_WARNING_END-->
+<!--DEV_DOCS_WARNING_START-->
+The following documentation refers to the in-development version of `splunk-otel-java`. Docs for the latest version ([v1.14.0](https://github.com/signalfx/splunk-otel-java/releases/latest)) can be found [here](https://github.com/signalfx/splunk-otel-java/blob/v1.14.0/README.md).
+
```

Comment:
This fragment is duplicated. I think, if I remember correctly, I always run both pre- and post-release scripts on a patch release; the first one to bump the version to the patch version everywhere, the second one to add that block of text back.